### PR TITLE
refactor: Convert ImportTest to Kotlin

### PR DESCRIPTION
**Pull request by david-allison**

### CODE REVIEW

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/libanki/ImportTest.kt [hunk 1]
```diff
@@ -13,256 +13,244 @@
...
-import androidx.test.filters.SdkSuppress;
-import androidx.test.rule.GrantPermissionRule;
-
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.contains;
-import static org.hamcrest.Matchers.hasSize;
-import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assume.assumeThat;
```
Comment (BrayanDSO) [inline AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/libanki/ImportTest.kt]: import these back

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/libanki/ImportTest.kt [hunk 2]
```diff
@@ -13,256 +13,244 @@
...
+    var retry = RetryRule(10)
     @Before
-    public void setUp() throws IOException {
-        mTestCol = getEmptyCol();
+    @Throws(IOException::class)
+    fun setUp() {
+        mTestCol = emptyCol
         // the backend provides its own importing methods
-        assumeThat(BackendFactory.getDefaultLegacySchema(), is(true));
+        Assume.assumeThat(defaultLegacySchema, Matchers.`is`(true))
```
Comment (BrayanDSO) [inline AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/libanki/ImportTest.kt:67]: `is` -> equalTo

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/libanki/ImportTest.kt [hunk 3]
```diff
@@ -13,256 +13,244 @@
...
+import org.junit.*
+import org.junit.runner.RunWith
+import java.io.File
+import java.io.FileOutputStream
+import java.io.IOException
+import java.util.*
+
+@RunWith(AndroidJUnit4::class)
+class ImportTest : InstrumentedTest() {
+    private var mTestCol: Collection? = null
```
Comment (BrayanDSO) [inline AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/libanki/ImportTest.kt:43]: lateinit

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/libanki/ImportTest.kt [hunk 4]
```diff
@@ -13,256 +13,244 @@
...
+import org.hamcrest.Matchers
+import org.junit.*
+import org.junit.runner.RunWith
+import java.io.File
+import java.io.FileOutputStream
+import java.io.IOException
+import java.util.*
+
+@RunWith(AndroidJUnit4::class)
+class ImportTest : InstrumentedTest() {
```
Comment (BrayanDSO) [inline AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/libanki/ImportTest.kt:42]: IDE lint

**Review by BrayanDSO — CHANGES_REQUESTED** [review]
for the other reviewers: the CSV tests are going away soon, so no need to create cleanups on them

**Review by BrayanDSO — APPROVED** [review]

**Review by lukstbit — APPROVED** [review]

**Pull request merged**